grammar.js

```diff
@@ -1787,18 +749,14 @@ module.exports = grammar({
             )
         ),
 
-        // TODO: Comment scanner code
         ranged_tag_content: $ =>
```

Comment:
The `ranged_tag_content` and `ranged_verbatim_tag_content` still need some work. Especially they need to be handled properly by the injections in https://github.com/nvim-neorg/neorg/pull/337 where a trivial change of `@table` to `#table` does not result in the desired effect as shown below:
```norg
#table
*bold heading* | /italic heading/
some value | _some underlined value_
#end
```
leads to an AST when the injection is updated:
```
ranged_tag [0, 0] - [3, 4]
  name: tag_name [0, 1] - [0, 6]
    tag_name_element [0, 1] - [0, 6]
  content: ranged_tag_content [1, 0] - [3, 0]
    row [1, 0] - [2, 0]
      cell [1, 0] - [1, 14]
        paragraph [1, 0] - [1, 14]
          paragraph [1, 0] - [1, 14]
            paragraph_segment [1, 0] - [1, 14]
              bold [1, 0] - [1, 14]
      cell [1, 17] - [1, 33]
        paragraph [1, 17] - [1, 33]
          paragraph [1, 17] - [1, 33]
            paragraph_segment [1, 17] - [1, 33]
              italic [1, 17] - [1, 33]
    row [2, 0] - [3, 0]
      cell [2, 0] - [2, 10]
        paragraph [2, 0] - [2, 10]
          paragraph [2, 0] - [2, 10]
            paragraph_segment [2, 0] - [2, 10]
      cell [2, 13] - [2, 36]
        paragraph [2, 13] - [2, 36]
          paragraph [2, 13] - [2, 36]
            paragraph_segment [2, 13] - [2, 36]
              underline [2, 13] - [2, 36]
    paragraph_segment [1, 0] - [1, 33]
      bold [1, 0] - [1, 14]
      italic [1, 17] - [1, 33]
    paragraph_segment [2, 0] - [2, 36]
      underline [2, 13] - [2, 36]
  ranged_tag_end [3, 0] - [3, 4]
```
We need to either: remove the nested injection of `norg` inside of `norg_table` but this will still treat the entire table as a `paragraph` from the `norg` parser perspective..
Or figure out a better solution..